Codeinwp/templates-patterns-collection
# Release

- Update dependencies
- Fix: Modal pagination styles
- Feat: Premade templates automatic screenshot generation
- Fix: Individual bulk import

### Testing instructions
1. Check that the import works as before
2. Check you can save an import templates
3. Check templates listing and premade templates listing work as expected.
